src/QsCompiler/Tests.LanguageServer/Tests.LanguageServer.csproj

```diff
@@ -26,6 +26,9 @@
     <ProjectReference Include="..\LanguageServer\LanguageServer.csproj">
       <Private>true</Private>
     </ProjectReference>
+    <ProjectReference Include="..\..\QsFmt\App\App.fsproj">
+      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+    </ProjectReference>
```

Comment:
Is this supposed to be part of this PR?

Reply:
Kind of - I was noticed and was bothered by the solution not building properly, and it seemed silly to create a separate PR.